### ISSUE TITLE
Use Helm 3/remove Tiller?

### ISSUE THREAD

**Issue by mogul** [gruntwork-io/terraform-google-gke#82]
Thanks so much for providing [this article](https://cloud.google.com/blog/products/devops-sre/deploying-a-production-grade-helm-release-on-gke-with-terraform) and the accompanying repo! Our team needs to deploy a single Helm chart in GKE with as few moving parts as possible to help us with federal compliance, and this is very instructive.

In the time since the article was written, [Helm 3 dropped the use of Tiller](https://helm.sh/docs/faq/#removal-of-tiller), further simplifying and integrating with k8s. Do you have any plans to update your repo to make use of Helm 3 and drop the Tiller component?

**Comment by robmorgan** [comment]
Hi @mogul,

Thanks for your comment.

We're hoping to prioritize Helm 3 support soon in the next few months. It also requires changes to some of the tooling we depend on such as [helm-kubernetes-services](https://github.com/gruntwork-io/helm-kubernetes-services) and [terratest](https://github.com/gruntwork-io/terratest) (our open-source testing framework). 

The good news is one of our engineers has started to tackle this a bit earlier than expected. See:
- https://github.com/gruntwork-io/terratest/pull/467
- https://github.com/gruntwork-io/helm-kubernetes-services/pull/62

So basically when those PRs are released, we would be at a point to start investigating the changes required to our open-source GKE module.

Regards,

Rob

**Comment by mogul** [comment]
Great to hear it! We'll keep our eyes peeled and try working with the Helm 2 example in the meantime.
